samples/oauth_rails_vethealth/test/controllers/session_controller_test.rb

```diff
@@ -0,0 +1,7 @@
+require 'test_helper'
+
+class SessionControllerTest < ActionDispatch::IntegrationTest
+  # test "the truth" do
```

Comment:
Some tests for the controllers would be good. You have some special cases for the session info and stuff (I think?) that would nice to capture here